backend/tests/test_endpoints_moodle.py

```diff
@@ -409,47 +409,57 @@ def test_get_course_build_by_academic_year(
+        "academic_year": "AY 2025",
+        "academic_year_short": "AY25",
+        "status": "created",
+        "creator_email": "manager@rice.edu",
+    }
+
+    expected_data_2 = {
+        "instructor_firstname": "Leon",
+        "instructor_lastname": "Simmons",
+        "instructor_email": "lsimmons@rice.edu",
+        "course_name": "Algebra 1 - Leon Simmons (AY 2025)",
+        "course_shortname": "Alg1 LS AY25",
+        "course_id": None,
+        "course_enrollment_url": None,
+        "course_enrollment_key": None,
+        "school_district_name": school_district_name,
+        "academic_year": "AY 2025",
+        "academic_year_short": "AY25",
+        "status": "created",
+        "creator_email": "manager@rice.edu",
+    }
+
+    expected_data_3 = {
+        "instructor_firstname": "Reed",
+        "instructor_lastname": "Thompson",
+        "instructor_email": "lthompson@rice.edu",
+        "course_name": "Algebra 1 - Reed Thompson (AY 2025)",
+        "course_shortname": "Alg1 RT AY25",
+        "course_id": 47,
+        "course_enrollment_url": "url.com",
+        "course_enrollment_key": "12345",
+        "school_district_name": school_district_name,
+        "academic_year": "AY 2025",
+        "academic_year_short": "AY25",
+        "status": "created",
+        "creator_email": "manager@rice.edu",
+    }
 
-    assert data[1].get("instructor_firstname") == "Leon"
-    assert data[1].get("instructor_lastname") == "Simmons"
-    assert data[1].get("instructor_email") == "lsimmons@rice.edu"
-    assert data[1].get("course_name") == "Algebra 1 - Leon Simmons (AY 2025)"
-    assert data[1].get("course_shortname") == "Alg1 LS AY25"
-    assert data[1].get("course_id") is None
-    assert data[1].get("course_enrollment_url") is None
-    assert data[1].get("course_enrollment_key") is None
-    assert data[1].get("school_district_name") == school_district_name
-    assert data[1].get("academic_year") == "AY 2025"
-    assert data[1].get("academic_year_short") == "AY25"
-    assert data[1].get("status") == "created"
-    assert data[1].get("creator_email") == "manager@rice.edu"
-
-    assert data[2].get("instructor_firstname") == "Reed"
-    assert data[2].get("instructor_lastname") == "Thompson"
-    assert data[2].get("instructor_email") == "lthompson@rice.edu"
-    assert data[2].get("course_name") == "Algebra 1 - Reed Thompson (AY 2025)"
-    assert data[2].get("course_shortname") == "Alg1 RT AY25"
-    assert data[2].get("course_id") == 47
-    assert data[2].get("course_enrollment_url") == "url.com"
-    assert data[2].get("course_enrollment_key") == "12345"
-    assert data[2].get("school_district_name") == school_district_name
-    assert data[2].get("academic_year") == "AY 2025"
-    assert data[2].get("academic_year_short") == "AY25"
-    assert data[2].get("status") == "created"
-    assert data[2].get("creator_email") == "manager@rice.edu"
+    assert any(expected_data_1.items() <= item.items() for item in data)
```

Comment:
Is there any reason why we couldn't use the stricter `==` on all of these assertions?